source/gra3-uni4/lec-interpretarExpresionesDivision-PP1.ptx

```diff
@@ -2,7 +2,7 @@
 <exercise xmlns:xi="http://www.w3.org/2001/XInclude">
 
 <statement>
-  <p>Hay 20 pupitres en la clase. Están divididos equitativamente en 5 grupos. ¿Cuántos pupitres hay en cada grupo?</p>
+  <p>Hay 20 pupitres en la clase. Están separados en 5 grupos igual de grandes. ¿Cuántos pupitres hay en cada grupo?</p>
```

Comment:
El original es 
> There are 20 desks in the class. They are divided equally into 5 groups

No me parece que sea buena idea hacer este cambio. 

GLO tiene:
> divided equally
(be) divided equally

--->
> • repartidos equitativamente
• repartidos en partes iguales
En contextos de división:
• dividir equitativamente"

Reply:
Creo que mejor revertir este @veronicamariGNo

Reply:
Volví al divided, pero no puse "equitativamente" pues no sale en las lecciones 1 a 5. EAJ estuvo de acuerdo.